spec/intl.html

```diff
@@ -1685,20 +1685,64 @@ <h1>
           </dl>
         </emu-clause>
 
-        <emu-clause id="sec-temporal-calendardatemergefields" type="abstract operation">
+        <emu-clause id="sec-temporal-calendarkeystoexclude" type="abstract operation">
           <h1>
-            CalendarDateMergeFields (
+            CalendarKeysToExclude (
               _calendar_: a String,
-              _fields_: an Object,
-              _additionalFields_: an Object,
-            ): an Object
+              _key_: a property key,
+            ): a List of property keys
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It takes two Objects of calendar-specific fields for the calendar represented by _calendar_ in _fields_ and _additionalFields_ and returns a new Object that includes both sets of fields. The values in _additionalFields_ should supersede the values in _fields_. Also, the returned Object must be free of ambiguity or conflicts. This is relevant for calendars which accept fields other than the standard set of built-in calendar fields.</dd>
+            <dd>
+              It performs the calendar-specific operation of determining which fields are exclusive for the field given by _key_.
+              The returned List may be empty.
+            </dd>
           </dl>
+          <p>
+            Exclusive fields means the set of fields that changing the field named _key_ can potentially conflict with or invalidate.
+            This operation is relevant for calendars which accept fields other than the standard set of ISO calendar fields, in order to implement the Temporal objects' `with()` methods, and `Temporal.Calendar.prototype.mergeFields()` in such a way that the result is free of ambiguity or conflicts.
```

Comment:
```suggestion
            This operation is relevant for calendars which accept fields other than the standard set of ISO calendar fields, in order to implement `Temporal.Calendar.prototype.mergeFields()` in such a way that the result is free of ambiguity or conflicts.
```

Reply:
I think including the phrase about `with()` here makes it clearer how this ultimately relates to user code.